packages/driver/test/cypress/integration/commands/actions/click_spec.js

```diff
@@ -266,7 +268,10 @@ describe('src/cy/commands/actions/click', () => {
     it('will not send mouseEvents/focus if pointerdown is defaultPrevented', () => {
       const $btn = cy.$$('#button')
 
+      // let clicked = false
```

Comment:
these are all for debugging purposes, you can comment out 271, 274, and 283 to run the test with a manual click

Reply:
ah, disregard my comments then, it would be nice if this was commented somewhere in the file but they are in so many places that would still be hard to discover